SPECS/ceph/ceph.spec

```diff
@@ -1,16 +1,18 @@
 #disable debuginfo because ceph-debuginfo rpm is too large
 %define debug_package %{nil}
-%global _python_bytecompile_extra 1
-
+%define _unpackaged_files_terminate_build 0 
```

Comment:
Why are we leveraging this option ?

Reply:
its produces few unpackaged files, so added this so that we dont exit in the build process.